front/app/utils/cl-intl/index.tsx

```diff
@@ -7,3 +8,8 @@ import useIntl from './useIntl';
 
 export { FormattedMessage, injectIntl, useIntl, useFormatMessageWithLocale };
 export type { MessageDescriptor };
+
+export const shouldCapitilize = (locale: Locale) => {
+  const localesNotCapitlized = ['de-DE'];
+  return !localesNotCapitlized.includes(locale);
+};
```

Comment:
I believe we've run into this before for other strings on the platform, so I thought we could add a util to check? :thinking: Then we can add any other languages which might have this issue as well in the future.

Open to a different suggestion though as well!

Reply:
E.g. I think we might want this for German for the "[x ideas](https://go-vocal.slack.com/archives/C07PEAULUNS/p1733482932325089)" change, though I'll need to check with Jelena :thinking:

Reply:
Or we could rename this to "shouldForceCapitalize" or something? I guess that's more what this does..

The problem is that the `capitalize` function from lodash capitalizes the first letter of the string, but then makes all other letters lowercase (which is incorrect behaviour for German) :thinking: